support-frontend/assets/pages/digital-subscription-checkout/thankYouContent.jsx

```diff
@@ -2,33 +2,44 @@
 
 // ----- Imports ----- //
 
-import React from 'react';
-import { connect } from 'react-redux';
+import * as React from 'react';
 
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
 
 import Content from 'components/content/content';
 import Text, { LargeParagraph } from 'components/text/text';
-import MarketingConsent from 'components/subscriptionCheckouts/thankYou/marketingConsentContainer';
```

Comment:
I'm having trouble following why we're now passing `marketingConsent` as prop into `thankYouContent.jsx` and `thankYouPendingContent.jsx` now rather than importing as before?

Reply:
Because we have to map from a different redux state to the marketing consent props depending on where the control is used.